pkg/adaptation/adaptation.go

```diff
@@ -335,26 +335,53 @@ func (r *Adaptation) startPlugins() (retErr error) {
 		}
 	}()
 
+	var errs []error
 	for i, name := range names {
-		log.Infof(noCtx, "starting plugin %q...", name)
+		log.Infof(noCtx, "starting pre-installed NRI plugin %q...", name)
 
 		id := ids[i]
-
 		p, err := r.newLaunchedPlugin(r.pluginPath, id, name, configs[i])
 		if err != nil {
-			return fmt.Errorf("failed to start NRI plugin %q: %w", name, err)
+			errs = append(errs, fmt.Errorf("[%s] %w", name, err))
+			log.Warnf(noCtx, "failed to initialize pre-installed NRI plugin %q: %w", name, err)
```

Comment:
With `log.Warnf()` we should use `%v` (value in default format) instead of `%w` (wrap error, but only understood by `fmt.Errorf()`).